openverse_catalog/dags/common/slack.py

```diff
@@ -224,7 +224,11 @@ def send_message(
     """Send a simple slack message, convenience message for short/simple messages."""
     if not should_send_message(http_conn_id):
         return
-    s = SlackMessage(username, icon_emoji, http_conn_id=http_conn_id)
+
+    environment = Variable.get("environment", default_var="dev")
```

Comment:
We are setting the environment to "dev" by default [elsewhere in this file](https://github.com/WordPress/openverse-catalog/blob/main/openverse_catalog/dags/common/slack.py#L245) so I copied it.

[In the API](https://github.com/WordPress/openverse-api/blob/main/ingestion_server/ingestion_server/slack.py#L29) we explicitly use `local`, and I believe `dev` _ought_ to refer to `staging`. Would love some feedback on what to use here in the Catalog 🤔

Reply:
Ah, yes - this is all over the place :sweat_smile: We have a ticket for unifying them across the board: https://github.com/WordPress/openverse-api/issues/478

For now I think keep the standards that already exist within a file/repo is solid until we tackle that ticket!